src/warning/warning-printer.ts

```diff
@@ -120,7 +121,7 @@ export class WarningPrinter {
   private async _getLinesOfText(
       startLine: number, endLine: number, localPath: string) {
     const contents = await this._options.analyzer.load(localPath);
-    return contents.split('\n').slice(startLine, endLine + 1);
+    return contents.split(EOL).slice(startLine, endLine + 1);
```

Comment:
I've dropped standalone `\r` from EOL case (since its archaic: Mac OS 9 and earlier)

Would you still disagree then?